components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I copied the overlay changes from #1509. Can we create a separate PR to add them since more than one component depends on overaly?

Reply:
Should Overlay than be it's own component like Underlay?: https://github.com/adobe/spectrum-css/blob/main/components/underlay/index.css

Reply:
Hmm, I'm not sure, but perhaps this could be done separately to not impact Popover in #1509.

Reply:
This would be the easiest way to share a component / same code across multiple components if I understand your comment correct. 

> I copied the overlay changes from https://github.com/adobe/spectrum-css/pull/1509. Can we create a separate PR to add them since more than one component depends on overaly @pfulton?

Reply:
I removed overlay related styles since overlay will likely updated to be a part of Modal.